### ISSUE TITLE
Latest 15 Papers - May 08, 2025

### ISSUE THREAD

**Issue by KMnO4-zx** [KMnO4-zx/DailyArXiv#20]
**Please check the [Github](https://github.com/KMnO4-zx/DailyArXiv) page for a better reading experience and more papers.**

## Geoscience
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[GeoRSMLLM: A Multimodal Large Language Model for Vision-Language Tasks in Geoscience and Remote Sensing](http://arxiv.org/abs/2503.12490v1)** | 2025-03-16 | [papers-cool](https://papers.cool/arxiv/2503.12490v1) |  |
| **[Towards Foundation Models: Evaluation of Geoscience Artificial Intelligence with Uncertainty](http://arxiv.org/abs/2501.14809v1)** | 2025-01-15 | [papers-cool](https://papers.cool/arxiv/2501.14809v1) |  |
| **[When Geoscience Meets Foundation Models: Towards General Geoscience Artificial Intelligence System](http://arxiv.org/abs/2309.06799v5)** | 2024-11-12 | [papers-cool](https://papers.cool/arxiv/2309.06799v5) | <details><summary>accpe...</summary><p>accpeted by IEEE Geoscience and Remote Sensing Magazine</p></details> |
| **[Prototype-Based Methods in Explainable AI and Emerging Opportunities in the Geosciences](http://arxiv.org/abs/2410.19856v1)** | 2024-10-22 | [papers-cool](https://papers.cool/arxiv/2410.19856v1) | <details><summary>Accep...</summary><p>Accepted at AI for Science Workshop-Oral (Attention Track), Proceedings of 41st International Conference on Machine Learning (ICML) 2024</p></details> |
| **[Interpretable Geoscience Artificial Intelligence (XGeoS-AI): Application to Demystify Image Recognition](http://arxiv.org/abs/2311.04940v2)** | 2024-05-07 | [papers-cool](https://papers.cool/arxiv/2311.04940v2) | <details><summary>there...</summary><p>there are some erros in the results, and a newer revision is still preparing</p></details> |
| **[GeoGalactica: A Scientific Large Language Model in Geoscience](http://arxiv.org/abs/2401.00434v2)** | 2024-04-13 | [papers-cool](https://papers.cool/arxiv/2401.00434v2) |  |
| **[A novel approach for estimating functions in the multivariate setting based on an adaptive knot selection for B-splines with an application to a chemical system used in geoscience](http://arxiv.org/abs/2306.00686v3)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2306.00686v3) | 29 pages, 27 figures |
| **[When Geoscience Meets Generative AI and Large Language Models: Foundations, Trends, and Future Challenges](http://arxiv.org/abs/2402.03349v1)** | 2024-01-25 | [papers-cool](https://papers.cool/arxiv/2402.03349v1) |  |
| **[Differentiable modeling to unify machine learning and physical models and advance Geosciences](http://arxiv.org/abs/2301.04027v2)** | 2023-12-27 | [papers-cool](https://papers.cool/arxiv/2301.04027v2) |  |
| **[K2: A Foundation Language Model for Geoscience Knowledge Understanding and Utilization](http://arxiv.org/abs/2306.05064v2)** | 2023-09-13 | [papers-cool](https://papers.cool/arxiv/2306.05064v2) |  |
| **[AI Security for Geoscience and Remote Sensing: Challenges and Future Trends](http://arxiv.org/abs/2212.09360v2)** | 2023-06-22 | [papers-cool](https://papers.cool/arxiv/2212.09360v2) |  |
| **[Coarse-to-fine Task-driven Inpainting for Geoscience Images](http://arxiv.org/abs/2211.11059v3)** | 2022-12-06 | [papers-cool](https://papers.cool/arxiv/2211.11059v3) |  |
| **[Investigating the fidelity of explainable artificial intelligence methods for applications of convolutional neural networks in geoscience](http://arxiv.org/abs/2202.03407v2)** | 2022-09-05 | [papers-cool](https://papers.cool/arxiv/2202.03407v2) |  |
| **[Data Centred Intelligent Geosciences: Research Agenda and Opportunities, Position Paper](http://arxiv.org/abs/2209.02384v1)** | 2022-08-20 | [papers-cool](https://papers.cool/arxiv/2209.02384v1) |  |
| **[Beyond Visuals : Examining the Experiences of Geoscience Professionals With Vision Disabilities in Accessing Data Visualizations](http://arxiv.org/abs/2207.13220v1)** | 2022-07-27 | [papers-cool](https://papers.cool/arxiv/2207.13220v1) |  |

## Spatial
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[ESTA: An Efficient Spatial-Temporal Range Aggregation Query Processing Algorithm for UAV Networks](http://arxiv.org/abs/2308.11977v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2308.11977v3) | 15 pages, 14 figures |
| **[LiLIS: Enhancing Big Spatial Data Processing with Lightweight Distributed Learned Index](http://arxiv.org/abs/2504.18883v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2504.18883v3) |  |
| **[Dual-Domain Masked Image Modeling: A Self-Supervised Pretraining Strategy Using Spatial and Frequency Domain Masking for Hyperspectral Data](http://arxiv.org/abs/2505.03220v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03220v1) | <details><summary>Prepr...</summary><p>Preprint to appear in IEEE IGARSS 2025</p></details> |
| **[Statistical Performance of Generalized Direction Detectors with Known Spatial Steering Vector](http://arxiv.org/abs/2505.03076v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03076v1) | <details><summary>4 pag...</summary><p>4 pages,2 figures.This manuscript is accepted by Signal Processing Letters</p></details> |
| **[Evaluating Multilevel Regression and Poststratification with Spatial Priors with a Big Data Behavioural Survey](http://arxiv.org/abs/2503.05915v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2503.05915v2) | <details><summary>36 pa...</summary><p>36 pages (13 main article + 23 pages of appendix), 4 figures. Prepared for submission to Journals of the Royal Statistical Society Series A</p></details> |
| **[Modeling Spatial Extremes using Non-Gaussian Spatial Autoregressive Models via Convolutional Neural Networks](http://arxiv.org/abs/2505.03034v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.03034v1) |  |
| **[Completing Spatial Transcriptomics Data for Gene Expression Prediction Benchmarking](http://arxiv.org/abs/2505.02980v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02980v1) | <details><summary>arXiv...</summary><p>arXiv admin note: substantial text overlap with arXiv:2407.13027</p></details> |
| **[SDA-GRIN for Adaptive Spatial-Temporal Multivariate Time Series Imputation](http://arxiv.org/abs/2410.03954v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2410.03954v2) |  |
| **[Smoothing of Headland Path Edges and Headland-to-Mainfield Lane Transitions Based on a Spatial Domain Transformation and Linear Programming](http://arxiv.org/abs/2407.05979v3)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2407.05979v3) | <details><summary>13 pa...</summary><p>13 pages, 12 figures, 4 tables</p></details> |
| **[Real-time Spatial Retrieval Augmented Generation for Urban Environments](http://arxiv.org/abs/2505.02271v1)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2505.02271v1) |  |
| **[Unaligned RGB Guided Hyperspectral Image Super-Resolution with Spatial-Spectral Concordance](http://arxiv.org/abs/2505.02109v1)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2505.02109v1) |  |
| **[Nonparametric Monitoring of Spatial Dependence](http://arxiv.org/abs/2408.17022v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2408.17022v2) | <details><summary>38 pa...</summary><p>38 pages, 9 figures, 19 tables</p></details> |
| **[Evaluation of Thermal Control Based on Spatial Thermal Comfort with Reconstructed Environmental Data](http://arxiv.org/abs/2505.00468v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2505.00468v2) |  |
| **[Discrete Spatial Diffusion: Intensity-Preserving Diffusion Modeling](http://arxiv.org/abs/2505.01917v1)** | 2025-05-03 | [papers-cool](https://papers.cool/arxiv/2505.01917v1) |  |
| **[FreeInsert: Disentangled Text-Guided Object Insertion in 3D Gaussian Scene without Spatial Priors](http://arxiv.org/abs/2505.01322v1)** | 2025-05-02 | [papers-cool](https://papers.cool/arxiv/2505.01322v1) |  |

## Time
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[THEMIS: Time, Heterogeneity, and Energy Minded Scheduling for Fair Multi-Tenant Use in FPGAs](http://arxiv.org/abs/2404.00507v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2404.00507v3) | <details><summary>12 Pa...</summary><p>12 Pages, 8 Figures, 3 Tables</p></details> |
| **[BRIDGE: Bootstrapping Text to Control Time-Series Generation via Multi-Agent Iterative Optimization and Diffusion Modelling](http://arxiv.org/abs/2503.02445v3)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2503.02445v3) | ICML 2025 |
| **[TTT: A Temporal Refinement Heuristic for Tenuously Tractable Discrete Time Reachability Problems](http://arxiv.org/abs/2407.14394v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2407.14394v2) | <details><summary>To ap...</summary><p>To appear in the proceedings of the American Control Conference (ACC) 2025</p></details> |
| **[Learning Symbolic Persistent Macro-Actions for POMDP Solving Over Time](http://arxiv.org/abs/2505.03668v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03668v1) | <details><summary>Accep...</summary><p>Accepted at 9th Conference on Neurosymbolic Learning and Reasoning</p></details> |
| **[Sylvester-Preconditioned Adaptive-Rank Implicit Time Integrators for Advection-Diffusion Equations with Variable Coefficients](http://arxiv.org/abs/2410.19662v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2410.19662v2) |  |
| **[DEFT: Differentiable Branched Discrete Elastic Rods for Modeling Furcated DLOs in Real-Time](http://arxiv.org/abs/2502.15037v5)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2502.15037v5) |  |
| **[The statistical spread of transmission outages on a fast protection time scale based on utility data](http://arxiv.org/abs/2407.15059v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2407.15059v2) |  |
| **[Real-Time Person Image Synthesis Using a Flow Matching Model](http://arxiv.org/abs/2505.03562v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03562v1) |  |
| **[Real-time small area estimation of food security in Zimbabwe: integrating mobile-phone and face-to-face surveys using joint multilevel regression and poststratification](http://arxiv.org/abs/2505.03517v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03517v1) |  |
| **[Efficient Multivariate Time Series Forecasting via Calibrated Language Models with Privileged Knowledge Distillation](http://arxiv.org/abs/2505.02138v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.02138v2) | <details><summary>Accep...</summary><p>Accepted by ICDE 2025</p></details> |
| **[Physics-informed neural network estimation of active material properties in time-dependent cardiac biomechanical models](http://arxiv.org/abs/2505.03382v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03382v1) |  |
| **[Time hierarchies for sublogarithmic-space quantum computation](http://arxiv.org/abs/2503.21582v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2503.21582v2) |  |
| **[The Unreasonable Effectiveness of Discrete-Time Gaussian Process Mixtures for Robot Policy Learning](http://arxiv.org/abs/2505.03296v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03296v1) | <details><summary>Submi...</summary><p>Submitted for publication to IEEE Transaction on Robotics</p></details> |
| **[Distilling Two-Timed Flow Models by Separately Matching Initial and Terminal Velocities](http://arxiv.org/abs/2505.01169v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.01169v2) |  |
| **[Rapid diagnostics of reconfigurable intelligent surfaces using space-time-coding modulation](http://arxiv.org/abs/2505.03266v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03266v1) | <details><summary>30 pa...</summary><p>30 pages, 6 figures, 1 table, supporting information</p></details> |

## Trajectory
| **Title** | **Date** | **KiMi** | **Comment** |
| --- | --- | --- | --- |
| **[DyTTP: Trajectory Prediction with Normalization-Free Transformers](http://arxiv.org/abs/2504.05356v2)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2504.05356v2) |  |
| **[Modeling Musical Genre Trajectories through Pathlet Learning](http://arxiv.org/abs/2505.03480v1)** | 2025-05-06 | [papers-cool](https://papers.cool/arxiv/2505.03480v1) | <details><summary>Adjun...</summary><p>Adjunct Proceedings of the 33rd ACM Conference on User Modeling, Adaptation and Personalization (UMAP Adjunct '25)</p></details> |
| **[Reward-Based Collision-Free Algorithm for Trajectory Planning of Autonomous Robots](http://arxiv.org/abs/2502.06149v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2502.06149v2) |  |
| **[Generative Trajectory Stitching through Diffusion Composition](http://arxiv.org/abs/2503.05153v2)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2503.05153v2) | <details><summary>Proje...</summary><p>Project page: https://comp-diffuser.github.io/</p></details> |
| **[Inverse Dynamics Trajectory Optimization for Contact-Implicit Model Predictive Control](http://arxiv.org/abs/2309.01813v3)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2309.01813v3) | <details><summary>IJRR ...</summary><p>IJRR accepted version</p></details> |
| **[Trajectory Minimum Touching Ball](http://arxiv.org/abs/2505.02472v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02472v1) |  |
| **[Riemannian Direct Trajectory Optimization of Rigid Bodies on Matrix Lie Groups](http://arxiv.org/abs/2505.02323v1)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2505.02323v1) | <details><summary>Accep...</summary><p>Accepted to Robotics: Science and Systems (RSS) 2025</p></details> |
| **[REPLAY: Modeling Time-Varying Temporal Regularities of Human Mobility for Location Prediction over Sparse Trajectories](http://arxiv.org/abs/2402.16310v4)** | 2025-05-05 | [papers-cool](https://papers.cool/arxiv/2402.16310v4) | <details><summary>Accep...</summary><p>Accepted by IEEE Transactions on Mobile Computing</p></details> |
| **[ARTEMIS: Autoregressive End-to-End Trajectory Planning with Mixture of Experts for Autonomous Driving](http://arxiv.org/abs/2504.19580v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.19580v2) |  |
| **[DiffOG: Differentiable Policy Trajectory Optimization with Generalizability](http://arxiv.org/abs/2504.13807v2)** | 2025-05-04 | [papers-cool](https://papers.cool/arxiv/2504.13807v2) |  |
| **[Leveraging Symmetry to Accelerate Learning of Trajectory Tracking Controllers for Free-Flying Robotic Systems](http://arxiv.org/abs/2409.11238v3)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2409.11238v3) | <details><summary>The f...</summary><p>The first three authors contributed equally to this work. This updated version reflects the final version to appear at IEEE International Conference on Robotics and Automation (ICRA) 2025</p></details> |
| **[Near-optimal Sensor Placement for Detecting Stochastic Target Trajectories in Barrier Coverage Systems](http://arxiv.org/abs/2505.00825v1)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2505.00825v1) | <details><summary>This ...</summary><p>This work is published in IEEE SysCon 2025</p></details> |
| **[Visual Trajectory Prediction of Vessels for Inland Navigation](http://arxiv.org/abs/2505.00599v1)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2505.00599v1) |  |
| **[ParkDiffusion: Heterogeneous Multi-Agent Multi-Modal Trajectory Prediction for Automated Parking using Diffusion Models](http://arxiv.org/abs/2505.00586v1)** | 2025-05-01 | [papers-cool](https://papers.cool/arxiv/2505.00586v1) |  |
| **[PPT: Pretraining with Pseudo-Labeled Trajectories for Motion Forecasting](http://arxiv.org/abs/2412.06491v2)** | 2025-04-30 | [papers-cool](https://papers.cool/arxiv/2412.06491v2) | <details><summary>18 pa...</summary><p>18 pages, 9 figures, updated results</p></details> |
